src/pyedb/dotnet/edb_core/materials.py

```diff
@@ -528,7 +528,7 @@ def add_material(self, name: str, **kwargs):
         -------
         :class:`pyedb.dotnet.edb_core.materials.Material`
         """
-        if name in self.__materials:
+        if name.lower() in [i.lower() for i in self.__materials]:
```

Comment:
@ring630 The refactoring of the Material classes removed the need to use `.lower()`. This is because it led with conflict associated to syslib materials and because users could name a material "MyMat" while having to reach it through "mymat". Is there a reason for using `lower()` here ?